src/components/ModelTableList/_model-table-list.scss

```diff
@@ -4,7 +4,8 @@
   margin-top: 0.5rem;
 }
 
-.model-table-list_error-message {
+.model-table-list_error-message a,
+.model-table-list_error-message a:visited {
```

Comment:
You should use the parent (&) selector here to reprevent duplication;

```css
.model-table-list_error-message {
  &, &:hover { .. }
}
```

Reply:
done